hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1162,6 +1178,14 @@
       The heartbeat interval from a Datanode to Recon.
     </description>
   </property>
+  <property>
+    <name>hdds.recon.initial.heartbeat.interval</name>
```

Comment:
`hdds.recon.heartbeat.interval.initial`

(Ideally these keys for Recon should have been `hdds.heartbeat.interval.recon` and `...recon.initial`, but `hdds.recon.heartbeat.interval` already exists since 1.3.0, so we should rename them with deprecation in separate task.)